Update notebooks (#2253)
influxdata/docs-v2: content/influxdb/cloud/notebooks/create-notebook.md

4. By default, a **Metric Selector** and **Visualization** cell appear (see [Cell types](#cell-types) below for details.)
5. Click the **+** icon to add a cell. See [Cell types](#cell-types) below for details on each type of cell.
6. Click **Preview** to preview the results of each cell in a raw data table without writing any data.
    {{% warn %}}

> If your cell contains a custom script that uses the `to()` function to write data, clicking **Preview** will write data.

Reviewer: ```suggestion
If your cell contains a custom script that uses any output function to write data to InfluxDB (the `to()` function) or send it to a 3rd party service, clicking **Preview** will write data.
```

> 6. Click **Preview** to preview the results of each cell in a raw data table without writing any data.
>   {{% warn %}}
>   If your cell contains a custom script that makes a call (for example, the `to()` function in Flux), clicking **Preview** might still write data.
>   {{% /warn %}}

Reviewer 2: - I think "might" should be "will".
- Does this get nested properly under the line item? You may have to format it like this:

```suggestion
6. Click **Preview** to preview the results of each cell in a raw data table without writing any data.
    {{% warn %}}
If your cell contains a custom script that uses the `to()` function to write data, clicking **Preview** will write data.
    {{% /warn %}}
```
